Formula/charmcraft.rb

```diff
@@ -83,24 +83,19 @@ class Charmcraft < Formula
     sha256 "9d643ff0a55b762d5cdb124b8eaa99c66322e2157b69160bc32796e824360e6d"
   end
 
-  resource "importlib-metadata" do
```

Comment:
Need to manually update resources as Homebrew's `pipgrip` is on `python@3.10` so it fails to add PyPI packages that have a Python version restriction for `python@3.9` or older.